redisenterprise/README.md

```diff
@@ -0,0 +1,104 @@
+# Redis Enterprise
+
+![img](./images/redis-enterprise.jpg)
+
+## Overview
+
+This integration provides [Redis Enterprise][1] monitoring and metrics for Datadog.
+
+### What is Redis Enterprise?
+
+[Redis Enterprise][1] is the fully supported mission critical ready version of Redis, the most loved database in the world.  In addition to the core open source Redis feature set, Redis Enterprise adds active-active geo-distribution, multi-model database features, enhanced observability and easier multi-tenancy management for higher uptimes.
+
+### Redis Enterprise Datadog Dashboard
+
+Redis Enterprise's Datadog integration provides a templated view across your clusters and databases allowing for operational insight unavailable in other products.  Understand usage patterns and plan for growth armed with the data necessary to make informed decisions.
+
+#### Database Overview
+![overview](./images/dashboard.png)
```

Comment:
Modify all links to be absolute path in the form:

```
https://raw.githubusercontent.com/DataDog/integrations-extras/master/redisenterprise/images/<IMAGE_NAME>
```

Reply:
@FlorianVeaux Updated to use absolute paths